include/sound/sof/ipc4/header.h

```diff
@@ -536,6 +536,7 @@ struct sof_ipc4_notify_resource_data {
 #define SOF_IPC4_DEBUG_SLOT_DEBUG_LOG		0x474f4c00 /* byte 0: core ID */
```

Comment:
@jsarha can you please elaborate a bit on what telemetry2 is, the need for it and how it is different from the existing telemetry?

Reply:
It tries to be what Liam has described here:
https://github.com/thesofproject/sof/issues/5521#issuecomment-1880945065

Reply:
yeah, but no one can figure this out. You have to add a simple enough abstract so that the PR is self-contained and self-explanatory.